src/Page/Object.elm

```diff
@@ -411,58 +433,62 @@ addItemButton db query =
-            , text "Ajouter un élément"
-            ]
+    button
+        [ class "btn btn-outline-primary w-100"
+        , class "d-flex justify-content-center align-items-center"
+        , class " gap-1 w-100"
+        , id "add-new-element"
+        , disabled <| firstAvailableProcess == Nothing
+        , onClick <|
+            case firstAvailableProcess of
+                Just process ->
+                    AddItem (Query.defaultItem process)
+
+                Nothing ->
+                    NoOp
+        ]
+        [ i [ class "icon icon-plus" ] []
+        , text "Ajouter un élément"
         ]
 
 
-itemListView : Db -> Definition -> Query -> List (Html Msg)
-itemListView db selectedImpact query =
+itemListView : Db -> Definition -> Results -> Query -> List (Html Msg)
+itemListView db selectedImpact results query =
     [ div [ class "card-header d-flex align-items-center justify-content-between" ]
         [ h2 [ class "h5 mb-0" ] [ text "Éléments" ] ]
-    , ul [ class "CardList list-group list-group-flush" ]
-        (if List.isEmpty query.items then
-            [ li [ class "list-group-item" ] [ text "Aucun élément" ] ]
-
-         else
-            case
-                query.items
-                    |> List.map (\{ amount, processId } -> ( amount, processId ))
-                    |> List.map (RE.combineMapSecond (Process.findById db.object.processes))
-                    |> RE.combine
-            of
-                Err error ->
-                    [ text error ]
-
-                Ok items ->
-                    List.map (itemView db selectedImpact) items ++ [ addItemButton db query ]
-        )
+    , div [ class "table-responsive" ]
+        [ table [ class "table mb-0" ]
+            [ tbody []
+                (if List.isEmpty query.items then
+                    [ tr [] [ td [] [ text "Aucun élément." ] ] ]
+
+                 else
+                    case
+                        query.items
+                            |> List.map (\{ amount, processId } -> ( amount, processId ))
+                            |> List.map (RE.combineMapSecond (Process.findById db.object.processes))
+                            |> RE.combine
+                    of
+                        Err error ->
+                            [ tr [] [ td [] [ text error ] ] ]
+
+                        Ok items ->
+                            Simulator.extractItems results
+                                |> List.map2 (itemView selectedImpact) items
+                )
+            ]
+        ]
+    , addItemButton db query
     ]
 
 
-itemView : Db -> Definition -> ( Query.Amount, Process ) -> Html Msg
-itemView db selectedImpact ( amount, process ) =
-    li [ class "list-group-item d-flex align-items-center gap-2" ]
-        [ div [ class "input-group w-33" ]
+itemView : Definition -> ( Query.Amount, Process ) -> Results -> Html Msg
+itemView selectedImpact ( amount, process ) itemResults =
+    tr []
```

Comment:
Maybe having a `<thead>` somewhere could ease the understanding the columns?

Reply:
I thought the data were rather self-explanatory? 
![image](https://github.com/user-attachments/assets/45a39cee-68e7-47a6-8f8f-7c693602b881)